installers/mac/build.sh

```
@@ -0,0 +1,117 @@
+
+
+# Check if the build was successful
+if [ -f "$OUTPUT_PKG" ]; then
+    print_info "Successfully created: $OUTPUT_PKG"
+    print_info "Package size: $(du -h "$OUTPUT_PKG" | cut -f1)"
+else
+    print_error "Failed to create pkg package"
+    exit 1
+fi
+
+rm -rf "$BALLERINA_TARGET"/*
```

Shellcheck (0.11.0)

[warning] Use "${var:?}" to ensure this never expands to /* .

(SC2115)